### DISCUSSION TITLE
Global worker registry

### DISCUSSION THREAD

**Discussion by harisvsulaiman** [agenda/agenda#1232] [Ideas]
We should have a registry to register agenda workers , jobs and job definitions.

Draft🤷‍♂️
https://github.com/agenda/agenda/pull/1229

**Comment by koresar** [comment]
Moving discussion from our Slack.

Haris suggests to have a UI tool (a-la Agendash but different) to monitor not just the database as the whole, but all workers. Kinda celery, but for node.js.
